.ci_support/build_all.py

```diff
@@ -68,9 +76,41 @@ def build_all(recipes_dir, arch):
                 if not built:
                     old_comp_folders.append(folder)
                     continue
+            if platform == 'osx' and (
+                    'MACOSX_DEPLOYMENT_TARGET' in text or
+                    'MACOSX_SDK_VERSION' in text):
+                config = safe_load(text)
+
+                if 'MACOSX_DEPLOYMENT_TARGET' in config:
+                    for version in config['MACOSX_DEPLOYMENT_TARGET']:
+                        version = tuple([int(x) for x in version.split('.')])
```

Comment:
Would the version have `#` in it if there are selectors?

Reply:
Does the YAML parser brings comments?

Reply:
Apparently not.